integration-tests/smoke/ocr_test.go

```diff
@@ -89,17 +51,25 @@ var _ = Describe("OCR Feed @ocr", func() {
 
 	DescribeTable("OCR suite on different EVM networks", func(
 		testNetwork *blockchain.EVMNetwork,
-		evmChart environment.ConnectedChart,
-		chainlinkCharts ...environment.ConnectedChart,
+		funding *big.Float,
 	) {
+		evmChart := ethereum.New(nil)
+		if !testNetwork.Simulated {
+			evmChart = ethereum.New(&ethereum.Props{
+				NetworkName: testNetwork.Name,
+				Simulated:   testNetwork.Simulated,
+				WsURLs:      testNetwork.URLs,
+			})
+		}
 		By("Deploying the environment")
 		testEnvironment = environment.New(&environment.Config{NamespacePrefix: "smoke-ocr"}).
 			AddHelm(mockservercfg.New(nil)).
 			AddHelm(mockserver.New(nil)).
-			AddHelm(evmChart)
-		for _, chainlinkChart := range chainlinkCharts {
-			testEnvironment.AddHelm(chainlinkChart)
-		}
+			AddHelm(evmChart).
+			AddHelm(chainlink.New(0, map[string]interface{}{
+				"env":      testNetwork.ChainlinkValuesMap(),
+				"replicas": 6,
```

Comment:
As opposed to previous implementation does it now always require 6 replicas?

Reply:
Ya, if you're running a basic OCR smoke test, you'll always do so with 6 replicas. If you want to use a different env setup for another test, you can just write another test, or make the environment a variable again.